obstacle_stop_planner_refine/design/class_diagram.pu

```diff
@@ -0,0 +1,43 @@
+@startuml
+
+namespace rclcpp {
+  class Node
+}
+
+namespace motion_planning {
+rclcpp.Node <|-- ObstacleStopPlannerNode
+ObstacleStopPlannerNode *-- AdaptiveCruiseController
+ObstacleStopPlannerNode *-- ObstacleStopPlannerDebugNode
+}
+
+@enduml
+
+@startuml
+
+namespace rclcpp {
+  class Node
+  class Logger
+}
+
+namespace motion_planning {
+rclcpp.Node <|-- ObstacleStopPlannerNode
+ObstacleStopPlannerNode *-- AdaptiveCruiseController
+ObstacleStopPlannerNode *-- ObstacleStopPlannerDebugNode
+ObstacleStopPlannerNode *-- ObstaclePointCloud
+ObstacleStopPlannerNode *-- VehicleInfo
+ObstaclePointCloud *-- VehicleInfo
+ObstaclePointCloud *-- rclcpp.Logger
+ObstacleStopPlannerNode *-- Trajectory
+AdaptiveCruiseController *-- rclcpp.Node
+ObstacleStopPlannerDebugNode *-- rclcpp.Node
+OneStepPolygon *-- VehicleInfo
+PointHelper *-- VehicleInfo
+
+
+vehicle_info_util.VehicleInfo <|-- VehicleInfo
+}
+namespace vehicle_info_util {
+class VehicleInfo
+}
+
+@enduml
```

Comment:
You don't have to create UML diagrams before I prepare the spec and document.
However, if it helps you understand the code, it's fine.

Reply:
I created a class diagram just to help understand myself and the reviewers. :+1: